include/miral/miral/fd_manager.h

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
This file is *not* for use by client code, so it should be in `src/miral`, not `include/miral`